src/transformers/visitors/orchestrationInternalFunctionCallVisitor.ts

```diff
@@ -331,29 +302,19 @@ FunctionCall: {
                 isCircuit = true;
               }
             }
-           if(callingfnDefIndicators[node.id]){
-             if(callingfnDefIndicators[node.id].isModified){
-               if(internalfnDefIndicators[node.id] && internalfnDefIndicators[node.id].isModified) {
-                 state.circuitImport ??= [];
-                 state.circuitImport.push('false');
+           if(callingfnDefIndicators[node.id] && internalfnDefIndicators[node.id] && internalfnDefIndicators[node.id].isModified){
+             if(callingfnDefIndicators[node.id].isModified)
                  isCircuit = false;
-                }
-              }
-             if(!callingfnDefIndicators[node.id].isModified) {
-               if(internalfnDefIndicators[node.id] && internalfnDefIndicators[node.id].isModified){
-                 state.circuitImport ??= [];
-                 state.circuitImport.push('true');
+             else
                  isCircuit = true;
```

Comment:
can we change it 
 if(internalfnDefIndicators[node.id] && internalfnDefIndicators[node.id].isModified){
            if(callingfnDefIndicators[node.id]) {
                  if(callingfnDefIndicators[node.id].isModified)
                    isCircuit = false;
                   else
                    isCircuit = true;
                }
              else
                isCircuit = true;